lib/constants.js

```diff
@@ -51,6 +51,8 @@ module.exports = {
     MQTTB_ALARM: 'MQTTB-ALARM',
     MQTT_DEFAULT_RETRIES: 5,
     MQTT_DEFAULT_RETRY_TIME: 5,
+    MQTT_SHARE_SUBSCRIPTION_GROUP: '$share/ul/',
+    MQTT_TOPIC_PROTOCOL: '/ul',
```

Comment:
Maybe documentation should be reviewed (MQTT parts) and the topics shown there adjusted? What about https://thinking-cities.readthedocs.io/en/master/device_api/index.html ?

Reply:
This PR only introduces a new way to publish; backward compatibility is preserved. We can describe new way in following PRs

Reply:
At the end documentatino changes were included. NTC